docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+using OrdinaryDiffEq, Random
+oprob = ODEProblem(rn, u0, (0.0, 10.0), p_true)
+sol = solve(oprob, Tsit5(); saveat=0.1)
+data = (0.8 .+ 0.4*rand(10)) .* sol[:P][10:10:end]
+
+# Plots the true solutions and the (synthetic data) measurements.
+using Plots
+plot(sol; idxs=:P, label="True solution")
+plot!(sol.t[10:10:end], data; label="Measurements", seriestype=:scatter)
+```
+
+Generally, PEtab takes five different inputs to define an optimisation problem (the minimiser of which generates a fitted parameter set):
+1. **Model**: The model which we want to fit to the data (a `ReactionSystem`).
+2. **Observables**: The possible observables that can be measured (a `Dict{String,PEtabObservable}`).
+3. **Estimation parameters**: The parameters which we want to fit to the data (a `Vector{PEtabParameter}`). 
+4. **Experimental (or simulation) conditions**: The simulations (each corresponding to a potential experiment) carried out during each step of the optimisation process (a `Dict{String,Dict{Symbol,Float64}}`).
+5. **Measurements**: The measurements to which the model is fitted (a `DataFrame`). 
+
+### Observables
+The observables define the quantities that we may measure in our experiments. Typically, each corresponds to a single species, however, [more complicated observables are possible](@ref petab_observables_observables). For each observable, we also need a noise formula, defining the uncertainty in its measurements. By default, PEtab assumes normally distributed noise, with a mean equal to the true value and a standard deviation which we have to define. It is also possible to use [more advanced noise formulas](@ref petab_observables_noise_formula).
+
+In our example, we only have a single possible observable, the `P` species. We will assume that the noise is normally distributed with a standard deviation `0.5` (in our case this is not true, however, typically the noise distribution is unknown and a guess have to be made). We combine this information in a `PEtabObservable` struct (to access the `P` species we must use [`@unpack`](@ref simulation_structure_interfacing_symbolic_representation)). Finally, we store all our observables in a dictionary, giving each an id tag (which is later used in the measurements input).
+
+```petab1
+@unpack P = rn
+obs_P = PEtabObservable(P, 0.5)
+observables = Dict("obs_P" => obs_P)
+nothing # hide
+```
+
+### Parameters
+Each parameter of the system can either be
+1. Known (described [here](@ref petab_parameters_known)).
+2. Depend on experimental/simulation conditions (described [here](@ref petab_simulation_conditions)). 
+3. Be an unknown that we wish to fit to data.
+
+In our case, we wish to fit all three system parameters (*kB*, *kD*, and *kP*). For each, we create a single `PEtabParameter`, and then gather these into a single vector.
+```petab1
+par_kB = PEtabParameter(:kB)
+par_kD = PEtabParameter(:kD)
+par_kP = PEtabParameter(:kP)
+params = [par_kB, par_kD, par_kP]
+nothing # hide
+```
```

Comment:
Think it should be nice to mention here that we have default bounds [lb, ub] = [1e-3, 1e]